api/spec/v1.yaml

```diff
@@ -2113,23 +2113,28 @@ components:
         verification:
           $ref: '#/components/schemas/RuntimeEvmContractVerification'
           description: |
-            Additional information obtained from contract verification. Only available for smart 
+            Additional information obtained from contract verification. Only available for smart
             contracts that have been verified successfully by Sourcify.
-    
+
     RuntimeEvmContractVerification:
       type: object
-      # properties:
-      #   abi:
-      #     type: object
-      #     description: |
-      #       The smart contract's [ABI](https://github.com/ethereum/wiki/wiki/Ethereum-Contract-ABI), in JSON format as defined by Solidity. Available if 
-      #       and only if the contract has been "verified" through [Sourcify](https://sourcify.dev), i.e. Sourcify has checked 
-      #       and confirmed that the source code and ABI correspond to the contract's creation bytecode.
-      #   compiler_version:
-      #     type: string
-      #     description: |
-      #       The version of the compiler used to verify the smart contract.
-      #     example: 0.8.18+commit.87f61d96
+      properties:
+        abi:
```

Comment:
I think it's cleaner to not include `abi` explicitly, since it's already present in `metadata_json`. It makes sense for the indexer to store the ABI separately _internally_ because it will use it frequently (re-parsing txs, events) and needs to be able to access it efficiently. But the Explorer will need to parse JSON either way, and doesn't care if the ABI is at `.abi` or at `.metadata_json.outputs.abi`.

I suggest we just add `Includes the smart contract's [ABI](https://docs.soliditylang.org/en/develop/abi-spec.html).` to the `metadata_json` description.